### PR TITLE
Ensure match reboot for autoyast test on  yast X86_64

### PR DESCRIPTION
**Pull request by hjluo**
##
### Ensure match reboot for autoyast test on yast X86_64

- Related ticket: https://progress.opensuse.org/issues/174736
- Needles: N/A
- Verification run: 
  - [**VR**](https://openqa.suse.de/tests/overview?version=15-SP7&build=hjluo_ay_yast_reboot&distri=sle)

##

### CODE REVIEW

File: tests/autoyast/installation.pm [hunk 1]
```diff
@@ -129,7 +129,7 @@ sub run {
     # Autoyast reboot automatically without confirmation, usually assert 'bios-boot' that is not existing on zVM
     # So push a needle to check upcoming reboot on zVM that is a way to indicate the stage done
     # For bsc#1231522, need match reboot popup on powerVM to reboot after installation.
-    push @needles, 'autoyast-stage1-reboot-upcoming' if is_s390x || (is_pvm && !is_upgrade && is_sle('<15-SP7')) || (check_var('VIRSH_VMM_FAMILY', 'xen') && check_var('VIRSH_VMM_TYPE', 'hvm'));
```
Comment (lemon-suse) [inline tests/autoyast/installation.pm:132]: Please add comments for the reason to exclude of hvm.

Comment (hjluo) [reply tests/autoyast/installation.pm:132]: Added comment there. please take a look again. thanks.

Comment (lemon-suse) [reply tests/autoyast/installation.pm:132]: This bug fix hasn't been backported yet, so should reserve the condition for <15-SP7.

Comment (hjluo) [reply tests/autoyast/installation.pm:132]: added and watit the VR to be run. thanks

Comment (hjluo) [reply tests/autoyast/installation.pm:132]: -[New VR](https://openqa.suse.de/tests/16313875#) passed, please take a look again. thanks!

**Review by lemon-suse — APPROVED** [review]

**Pull request merged**